examples/magento-graphcms/package.json

```diff
@@ -62,6 +62,7 @@
     "@graphcommerce/magento-graphql": "9.0.4-canary.10",
     "@graphcommerce/magento-graphql-rest": "9.0.4-canary.10",
     "@graphcommerce/magento-newsletter": "9.0.4-canary.10",
+    "@graphcommerce/magento-oci": "9.0.4-canary.10",
```

Comment:
You can load it with PRIVATE_ADDITIONAL_DEPENDENCIES configuration during deploy (configure on Vercel) or local development.